packages/components/src/button/src/Button.css

```diff
@@ -116,10 +116,17 @@ a.o-ui-button {
 
 /* SECONDARY */
 .o-ui-button-secondary {
-    box-shadow: 0 0 0 1px var(--o-ui-b-alias-high-break) inset;
+    border: 1px solid var(--o-ui-b-alias-high-break);
     color: var(--o-ui-text-alias-primary);
 }
 
+.o-ui-button-secondary:after {
+    top: -1px;
+    left: -1px;
+    right: -1px;
+    bottom: -1px;
+}
+
```

Comment:
Just curious about the context of that change. I guess it helps with the artifact issue but do we know what it does?

Reply:
Since we are using a border instead of a box shadow for outline components the focus style needs to move as a border takes space in the layout while a box shadow does not affect an element dimensions.